projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+import { Navigation } from '../../../helpers/navigation';
+import { CdsHelper } from '../../../helpers/vendor/cds/cds';
+import * as MercahndisingCarousel from '../../../helpers/vendor/cds/merchandising-carousel';
+import { ProfileTagHelper } from '../../../helpers/vendor/cds/profile-tag';
+describe('Profile-tag component', () => {
+  beforeEach(() => {
+    cy.server();
+    CdsHelper.setUpMocks();
+  });
+  it('Should wait for a user to accept consent before sending an event', () => {
```

Comment:
same case for the it clause/assertion, `Should -- should be in lowercase as well`